src/components/DataDimension/Detail.js

```diff
@@ -4,10 +4,16 @@ import InputLabel from '@material-ui/core/InputLabel'
 import MenuItem from '@material-ui/core/MenuItem'
 import Select from '@material-ui/core/Select'
 import i18n from '@dhis2/d2-i18n'
+import { TOTALS, DETAIL } from '../../modules/dataTypes'
 
 import { styles } from './styles/Details.style'
 
-export const Detail = ({ value, onDetailChange, detailAlternatives }) => (
+const getOptions = () => ({
```

Comment:
Moved this object from dataTypes to here. This was a case of over-engineering, as the Detail component is only used in this one case, where the options are Totals and details.